src/search/sr_optimizers.c

```
@@ -249,13 +278,15 @@ void sr_optimizer_config_apply(const sr_optimizer_config *cfg)
     sr_powell_iter_limit = cfg->max_iters;
     sr_sa_iter_limit = cfg->max_iters;
     sr_pso_iter_limit = cfg->max_iters;
+    sr_de_iter_limit = cfg->max_iters;
   } else {
```

Clang (14.0.6)

[note] else branch starts here

(clang)